tools/bump-version

```diff
@@ -0,0 +1,105 @@
+  .option('--deliverino-private-key <deliverino-private-key>', 'private key of the deliverino github app in pem format with base64 encoding')
+  .option('--owner <owner>', 'owner of the current repository, usually "Integreat"')
+  .option('--repo <repo>', 'the current repository, usually "integreat-webapp" or "integreat-react-native-app"')
+  .option('--branch <branch>', 'the current branch')
+
+const calculateNewVersion = versionFilePath => {
+  const versionFile = fs.readFileSync(versionFilePath)
+  // versionCode is just used in the integreat-react-native-app
+  const { versionName, versionCode } = JSON.parse(versionFile)
+  const versionNameParts = versionName.split('.')
+
+  const date = new Date()
+  const year = date.getFullYear()
+  const month = date.getMonth() + 1
+
+  const versionNameCounter = year === versionNameParts[0] && month === versionNameParts[1] ? versionNameParts[2] + 1 : 0
+  const newVersionName = `${year}.${month}.${versionNameCounter}`
+  const newVersionCode = versionCode ? versionCode + 1 : undefined
+
+  console.debug(`New version name: ${newVersionName}`)
+  if (newVersionCode) {
+    console.debug(`New version name: ${newVersionCode}`)
+  }
+
+  return {
+    versionName: newVersionName,
+    versionCode: newVersionCode
+  }
+}
+
+const commitVersionBump = async (newVersion, { versionFile, deliverinoPrivateKey, owner, repo, branch }) => {
+  const appId = 59249 // https://github.com/apps/deliverino
+  const privateKey = Buffer.from(deliverinoPrivateKey, 'base64').toString('ascii')
+
+  const octokit = new Octokit({ authStrategy: createAppAuth, auth: { id: appId, privateKey: privateKey } })
+  const { data: { id: installationId } } = await octokit.apps.getRepoInstallation({ owner, repo })
+  const { data: { token: installationToken } } = await octokit.apps.createInstallationToken({ installationId })
+
+  const appOctokit = new Octokit({ auth: installationToken })
+  const versionFileContent = await appOctokit.repos.getContents({ owner, repo, path: versionFile, ref: branch })
+
+  const { versionCode, versionName } = newVersion
+  const contentBase64 = Buffer.from(JSON.stringify(newVersion)).toString('base64')
+
+  const message = versionCode
+    ? `Bump version name to ${versionName} and version code to ${versionCode}\n[skip ci]`
+    : `Bump version name to ${versionName}\n[skip ci]`
+
+  await appOctokit.repos.createOrUpdateFile({
+    owner,
+    repo,
+    path: versionFile,
+    content: contentBase64,
+    branch,
+    message,
+    sha: versionFileContent.data.sha
+  })
+
+  console.debug(`Bumped version successfully commited with message "${message}"`)
+}
+
+program
+  .command('bump-version')
+  .description('bump the current versionName (and versionCode if available) and commit to github')
+  .action(async () => {
+    try {
+      const { dryRun, versionFile, deliverinoPrivateKey, owner, repo, branch } = program
+      console.log('versionFile', versionFile)
+      if (!dryRun) {
```

Comment:
You caan set these parameters as required in `commander`. That way you can remove this if.

Reply:
No, because those are not required if `dryRun` is true.

Reply:
Would be fine for me if you still have to pass the parameters. dry-run usually means to have the exactly same behaviour just without the persistent action.